website/docs/r/virtual_machine.html.markdown

```diff
@@ -335,9 +335,11 @@ For more information on the different example configurations, please check out t
 
 `storage_image_reference` supports the following:
 
-* `publisher` - (Required) Specifies the publisher of the image used to create the virtual machine. Changing this forces a new resource to be created.
-* `offer` - (Required) Specifies the offer of the image used to create the virtual machine. Changing this forces a new resource to be created.
-* `sku` - (Required) Specifies the SKU of the image used to create the virtual machine. Changing this forces a new resource to be created.
+* `image_id` - (Required, when using image resource) Specifies the ID of the (custom) image to use to create the virtual 
```

Comment:
We'd normally format this as follows:

- `foo` (Optional) Specifies the ID.... Cannot be set when `X`, `Y` and `Z` are set.

And then add an example to show usage of the image_id field?